services/jenkins-pipeline-monitor/handler.py

```diff
@@ -87,11 +95,21 @@ def filter_by_desired_release_job_type(latest_day_builds, desired_release_job_ty
 
 
 def status_check(builds):
-    for build in builds:
-        if build.get_status() == 'SUCCESS':
-            logging.info(f'Successful build {get_release_job_type(build)} {build.get_number()}')
-        else:
-            logging.info(f'Failure build {get_release_job_type(build)} {build.get_number()}')
+    """
+    Check the status of the filtered builds
+    If there is not a single build belong to desired release job type, log the failures
+    else check the status via Jenkins API and report accordingly
+    :param builds
+    """
+    if not builds:
```

Comment:
we are missing the case when `builds` have only one out of the two builds. In that case we should log one of the failure messages.

Reply:
Addressed